projects/plugins/jetpack/extensions/blocks/tiled-gallery/index.js

```diff
@@ -130,6 +130,14 @@ const blockAttributes = {
 			},
 		},
 	},
+	hideImageCaption: {
+		type: 'boolean',
+		default: true,
+	},
+	imageCrop: {
+		type: 'boolean',
+		default: true,
+	},
```

Comment:
Again here, this might need to be put in a native-specific file.

Reply:
From what I understand looking at the Gutenberg PR, `imageCrop` already existed as a [context](https://github.com/WordPress/gutenberg/blob/bee52e68292357011a799f067ad47aa1c1d710e1/docs/reference-guides/block-api/block-context.md) shared between the core Gallery and its inner core Image blocks.
I see you're adding it here as `true`, presumably because the square layout requires images to be cropped, [which translates to `resizeMode=cover`](https://github.com/WordPress/gutenberg/pull/35089/files#diff-7126ae03000a022bcefa0d2a5fb03b6aca56ed4047e3a49d2131babf229cf6c1R633).
While it wasn't intuitive what `imageCrop` did here, I think it's acceptable because there's no way to document these attributes (e.g. with a comment).
Looking ahead to when we add additional layouts (mosaic, columns, circles), I think this `resizeMode=cover` does make sense, so this feels like a solid approach 👍